Makefile

```
@@ -102,13 +102,17 @@ endif
 setup/moq:
 	GO111MODULE=off go get github.com/matryer/moq
 
+.PHONY: setup/service_account/oc_login
+setup/service_account/oc_login:
+	@oc login --token=$(shell sh -c "oc serviceaccounts get-token rhmi-operator -n ${NAMESPACE}") --server=$(shell sh -c "oc cluster-info | grep -Eo 'https?://[-a-zA-Z0-9\.:]*'") --kubeconfig=TMP_SA_KUBECONFIG --insecure-skip-tls-verify=true
```

Small suggestion but i think `oc whoami --show-server` can be used to directly get the cluster API url so might be cleaner than grepping from  `oc cluster-info` 

```suggestion
	@oc login --token=$(shell sh -c "oc serviceaccounts get-token rhmi-operator -n ${NAMESPACE}") --server=$(shell sh -c "oc whoami --show-server") --kubeconfig=TMP_SA_KUBECONFIG --insecure-skip-tls-verify=true
```

---

👍  thanks